src/AvoStorage.ts

```diff
@@ -6,6 +7,8 @@ export class AvoStorage {
 
   shouldLog: boolean;
   initialized = false;
```

Comment:
Previously initialization meant that data stored between sessions is restored. So basically if local storage is not available we can say that it's initialized. Would result in a new session though.